docs/simulators/Reef3D.md

```diff
@@ -70,6 +76,254 @@ task.wait()
+are important to understand before we configure our simulation run. These
+parameters are: ```N 41```, ```P 30 ``` and ```M 10```:
+
+### control.txt (for DiveMESH)
+```
+C 11 21
+C 12 21
+C 13 21
+C 14 21
+C 15 21
+C 16 21
+
+B 1 0.025
+B 10 0.0 2.0 0.0 1.0 0.0 1.0
+O 10 1.2 1.4 0.4 0.6 0.0 1.0
+
+M 10 4    <---- defines the nr. of processors for parallel computations (4)
+```
+
+### ctrl.txt (for Reef3D)
+```
+D 10 4
+D 20 2
+D 30 1
+F 30 3
+F 40 3
+F 54 0.5
+F 56 0.7
+N 40 3
+N 41 25.0    <---- set the maximum modeled time (25 seconds).
+N 45 50000
+N 47 0.2
+M 10 4    <---- defines the nr. of processors for parallel computations (4)
+P 10 1
+P 30 0.01    <---- defines the rate of paraview results (1 frame per 0.01 s)
+T 10 0
+W 22 -9.81
+```
+
+Observe that parameter ```M 10```, which controls the degree of parallism, is
+set to 4 threads/vCPUs, a very low number. Depending on the number of vCPUs we
+effectively wish to use for running the simulation, we will need to manually
+change  ```M 10``` on **both files** to match the specs of corresponding the VM. 
+Because this is already a reasoably heavy simulation, we will be using GCP VMs
+of the c3d family, supported by last-generation AMD chips. More specifically,
+we will be using ```c3d-standard-90``` machines with 90 vCPU. So, we will be
+setting ```M 10 90``` on both files.
+
+Also, Reef3D produces a huge amount of data. As it is currently configured, this
+simulation would produce several dozen gigabytes of data. To reduce that amount
+of data produced, we can reduce the maximum modeled time (```N 41```) and
+the rate at which Paraview data is being produced (```P 10```).
+
+So, for a reducing the amount of data generated, we will be reducing the rate
+at which Paraview information is going to be generated to "onyy" 25 frames per
+second, that is, we will set ```P 10 0.04```. 
+
+We are not going to reduce the maximum modeled time but, instead, we will add
+extra disk capacity to ensure we have enough space to write all the data will be
+generated. Therefore, we will request our machine to be equipped
+with **20GB** just for data, using the ```data_disk_gb``` parameter of the 
+```inductiva.resources.MachineGroup``` class.
+
+Here is the final script:
+
+```python
+import inductiva
+
+
+machine_group = inductiva.resources.MachineGroup(
+    machine_type="c3d-standard-90",
+    spot=True,
+    data_disk_gb=20)  ## We are adding extra disk space here
+machine_group.start()
+
+
+reef3d = inductiva.simulators.REEF3D()
+
+task = reef3d.run(input_dir="./10_2_3D_Dam_Break_with_Obstacle",
+                  on=machine_group,
+                  n_vcpus=90)   ## We need to make sure this matches `N 10`
+
+task.wait()
+task.download_outputs()
+
+## Make sure we terminate the machine.
+machine_group.terminate()
+```
+
+Running this script end-to-end should take about 25 minutes, distributed in
+the following way:
+
+* about 2 minutes in the preparation stage, including starting the executer VM.
```

Comment:
@sergiomsantos @lneves-ai @luispcunha : for now I am informally collecting these number. It would be great to have these number explicit in the stdout of my python script. These stages should be more explicit so that the user  (and ourselves) can understand more exactly where the time is being spent, and then make the necessary changes.

(cc @hpenedones )